grpc-interface-support/data/templates/python/ServiceImpl.py

```diff
@@ -0,0 +1,8 @@
+import grpc
+import ${{ package_id }}
+
+
+class ${{ service_name_camel_case }}Service${{ service_name_postfix }}(${{ package_id }}.${{ service_name }}${{ service_name_parent_postfix }}):
```

Comment:
Looks like a lot of variables. Maybe we just create one which contains the full class name? Makes it a lot more readable.

Reply:
I generally agree but I need a different `service_name_postfix` for the abstract parent class + impl class. Same with  `service_name_parent_postfix`. 

However I removed the `service_name_camel_case ` and just used `service_name`.